meta-dts-distro/recipes-dts/dts/dasharo-deploy/dasharo-deploy

```diff
@@ -104,13 +106,28 @@ SERIAL_NUMBER_DIR="/tmp/serial_number.txt"
 PROGRAMMER_BIOS="internal"
 PROGRAMMER_EC="ite_ec"
 
+DASHARO_VENDOR="3mdeb"
+DASHARO_NAME="Dasharo"
+
+look_for_dasharo() {
```

Comment:
@esesyn01 maybe let's call it `check_if_dasharo` and place it in some script with common functions

We could install in system script called `dts-functions.sh` which would not be executable but would only have common functions definitions. Later we could use it in `dts` or `dasharo-deploy` script by adding at the beggining

`source /path/to/dts-functions.sh`

WDYT about that?

Reply:
@TomaszAIR This is a good idea, but then, to be consistent, we should also move functions like `it5570_i2ec`, which is both in [dasharo-deploy](https://github.com/Dasharo/meta-dts/blob/disabling-flashing-when-dasharo-on-hw/meta-dts-distro/recipes-dts/dts/dasharo-deploy/dasharo-deploy#L41) and [ec_transition](https://github.com/Dasharo/meta-dts/blob/disabling-flashing-when-dasharo-on-hw/meta-dts-distro/recipes-dts/dts/dts/ec_transition#L16). Should I also deal with other scripts in this task or take care of `dasharo-deploy` and `dts`?